Marlin/src/gcode/config/M301.cpp

```diff
@@ -24,6 +24,7 @@
 
 #if ENABLED(PIDTEMP)
 
+#include "../../Marlin.h"
```

Comment:
If this is just to get the serial functions, then we can just include the serial header here.

Reply:
👍